frontends/bmd/src/pages/contest_page.tsx

```diff
@@ -107,67 +171,68 @@ export function ContestPage(): JSX.Element {
           updateVote={updateVote}
         />
       )}
-      <Sidebar
-        footer={
-          <React.Fragment>
-            <SettingsTextSize
-              userSettings={userSettings}
-              setUserSettings={setUserSettings}
-            />
-            <ElectionInfo
-              electionDefinition={electionDefinition}
-              ballotStyleId={ballotStyleId}
-              precinctSelection={singlePrecinctSelectionFor(precinctId)}
-              horizontal
-            />
-          </React.Fragment>
-        }
-      >
-        <Prose>
-          <Text center>
-            This is the <strong>{ordinal(currentContestIndex + 1)}</strong> of{' '}
-            {pluralize('contest', contests.length, true)}.
-          </Text>
-          {isReviewMode ? (
-            <p>
-              <LinkButton
-                large
-                primary={isVoteComplete}
-                to={`/review#contest-${contest.id}`}
-                id="next"
-              >
-                <TextIcon arrowRight white={isVoteComplete}>
-                  Review
-                </TextIcon>
-              </LinkButton>
-            </p>
-          ) : (
+      {isPortrait ? (
+        <ButtonFooter>
+          {isReviewMode ? ReviewScreenButton : NextContestButton}
+          {!isReviewMode && PreviousContestButton}
+          {/* <Button large onPress={() => {}} aria-label="Change Language">
+            English
+          </Button> */}
+          {/* <SegmentedButton>
+            <Button
+              large
+              primary
+              onPress={() => {}}
+              aria-label="Change Language"
+            >
+              Español
+            </Button>
+            <Button large onPress={() => {}} aria-label="Change Language">
+              English
+            </Button>
+          </SegmentedButton> */}
```

Comment:
Should this comment be removed?

Reply:
No, I had thought to leave this in as example of how to add language. Thoughts?